controllers/suite_test.go

```diff
@@ -336,15 +336,15 @@ func setUpTestProxy() {
 	}
 
 	err = k8sClient.Create(ctx, &testProxyService)
-	Expect(err != nil || errors.IsConflict(err))
+	Expect(err).NotTo(HaveOccurred())
```

Comment:
The test-suite can skip error if the test proxy service already exist:
```suggestion
	Expect(err == nil || errors.IsConflict(err))
```